### PR TITLE
Add Metadata Element page for current metadata key/value/behavior

### PR DESCRIPTION
**Pull request by sgchoi5**
Added Metadata Element page at this position 
...
- Manifest Text Editor
- Additional Manifest Elements
...

### CODE REVIEW

File: docs/application/toc_all.md [hunk 1]
```diff
@@ -1112,6 +1112,7 @@
 
 #### Manifest Editor
 ##### [Overview](/application/vstools/tools/manifest-editor.md)
+##### [Manifest Element](/application/vstools/tools/manifest-element.md)
```
Comment (journey2w) [inline docs/application/toc_all.md:1115]:
```suggestion
##### [Manifest Element](/application/vstools/tools/metadata-element.md)
```

File: docs/application/toc_all.md [hunk 2]
```diff
@@ -1174,6 +1175,7 @@
 #### Configure Your App
 ##### [Overview](/application/tizen-studio/native-tools/configuring.md)
 ##### [Manifest Text Editor](/application/tizen-studio/native-tools/manifest-text-editor.md)
+##### [Manifest Element](/application/tizen-studio/native-tools/manifest-element.md)
```
Comment (journey2w) [inline docs/application/toc_all.md:1178]:
```suggestion
##### [Manifest Element](/application/tizen-studio/native-tools/metadata-element.md)
```

**Review by journey2w — CHANGES_REQUESTED** [review]

**Comment by sgchoi5** [comment]
Fixed filename.
Now, it works correctly.

https://docs1.stg.tizen.org/staging/1681/application/vstools/tools/metadata-element/
https://docs1.stg.tizen.org/staging/1681/application/tizen-studio/native-tools/metadata-element/

File: docs/application/toc_all.md (hunk 2 above)
Comment (sgchoi5) [reply docs/application/toc_all.md]: fixed title and filename. manifest(x) => metadata (o)

File: docs/application/tizen-studio/native-tools/metadata-element.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Metadata element
```
Comment (safir-srbd) [inline docs/application/tizen-studio/native-tools/metadata-element.md:1]: First headings should follow Title Case.
```suggestion
# Metadata Element
```

File: docs/application/vstools/tools/metadata-element.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Metadata element
```
Comment (safir-srbd) [inline docs/application/vstools/tools/metadata-element.md:1]: First headings should follow Title Case.
```suggestion
# Metadata Element
```

**Review by safir-srbd — COMMENTED** [review]
@sgchoi5 Hello. Hope you are doing good. I have gone through the PR and have suggested a few changes. Please incorporate them when you can.

File: docs/application/tizen-studio/native-tools/metadata-element.md (hunk 3 above)
Comment (sgchoi5) [reply docs/application/tizen-studio/native-tools/metadata-element.md]: thanks! fixed
https://docs1.stg.tizen.org/staging/1681/application/tizen-studio/native-tools/metadata-element/

File: docs/application/vstools/tools/metadata-element.md (hunk 4 above)
Comment (sgchoi5) [reply docs/application/vstools/tools/metadata-element.md]: thanks! fixed
https://docs1.stg.tizen.org/staging/1681/application/tizen-studio/native-tools/metadata-element/

**Review by mijin85cho — COMMENTED** [review]
Hello. I skimmed the updates and got below suggestion.
Would you please check my comments?

File: docs/application/tizen-studio/native-tools/metadata-element.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+| 1   | http://tizen.org/metadata/allowedappid                   | 3.0   | Allowed application ids                                         | The service application can allow the application to launch                                  | The value is not defined                                                                                                                                                                                                                           | ```<metadata key="http://tizen.org/metadata/allowedappid" value="org.tizen.allowedapp"/>```                                                                             |
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/tizen-studio/native-tools/metadata-element.md]: thanks. disabled hyper link.

[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/tizen-studio/native-tools/metadata-element.md]: thanks! changed format from ``` to `

**Review by safir-srbd — APPROVED** [review]
@sgchoi5 Thank you for making all the changes suggested.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Comment by asm-foysal** [comment]
@journey2w hello Ms. Kang! Hope you're doing well.
Since you made the "change request" during your review, I'm not being able to merge this PR without your approval. 
Hence I'm dismissing the "change request", since it has already been addressed, and merging the PR.

**Comment by mijin85cho** [comment]
Hello, @asm-foysal 
I guess you are seeing "Merging is blocked" in the menu because of the requested changes.
I want to know if you can see the "Merge without waiting for requirements to be met (bypass branch protections)".
If you click the check box, I think you would be possible to click the "Merge pull request" button.

**Comment by asm-foysal** [comment]
@mijin85cho thank you for the advice, I've merged the PR.

**Pull request merged**